shardingsphere-distsql-parser/shardingsphere-distsql-parser-engine/src/main/java/org/apache/shardingsphere/distsql/parser/api/DistSQLStatementParserEngine.java

```diff
@@ -38,7 +39,11 @@ public SQLStatement parse(final String sql) {
         try {
             return new ResourceSQLStatementParserEngine().parse(sql);
         } catch (final ParseCancellationException | SQLParsingException ignored) {
-            return new RuleSQLStatementParserEngine().parse(sql);
+            try {
```

Comment:
Why do we need `catch` to parse `RuleSQLStatementParserEngine`?

Reply:
Maybe we can fefer to [ShardingSphereSQLParserEngine#parse0](https://github.com/apache/shardingsphere/blob/master/shardingsphere-infra/shardingsphere-infra-parser/src/main/java/org/apache/shardingsphere/infra/parser/ShardingSphereSQLParserEngine.java)